.github/workflows/check-i18n-changes.yml

```
@@ -0,0 +1,23 @@
+name: Protect content from manual changes
+
+on: 
+  pull_request:
+    paths:
+    - 'content/**'
```

This would allow changes to the English content like when adding a new version
```suggestion
    - 'content/**'
    - "!content/*/en-US/**"
```

---

> This would allow changes to the English content like when adding a new version

All content inside `content/` must be changed automatically with bots (@nodejs-crowdin and @github-actions).
[Workflow](https://github.com/nodejs/i18n/blob/b66ba115d520634cc91d10f4081f489cc0c5ef42/.github/workflows/nightly-sync.yml#L30)

---

It doesn't happen for new Node versions through the sync right now, it relies on the folder being adding in addtion to the package.json change

---

@nschonni if we are going to land #313 this wouldn't be a problem